installer.sh.in

```diff
@@ -1342,7 +1342,7 @@ menu_source() {
     DIALOG --title " Select installation source " \
         --menu "$MENULABEL" 8 70 0 \
         "Local" "Packages from ISO image" \
-        "Network" "Packages from official remote reposity"
+        "Network" "Base system only, downloaded from official reposity"
```

Comment:
repository*

Reply:
hah, I was glad at least to see I didn't write that.